### DISCUSSION TITLE
How do I optimize it, with frames dropping so badly

### DISCUSSION THREAD

**Discussion by lowking** [neovide/neovide#1474] [Q&A]
https://user-images.githubusercontent.com/33308659/184174149-10f6539a-4b44-4b72-8a51-15247dfa2e40.mov

I NEED HELP

**Comment by LamprosPitsillos** [comment]
Consider giving more information and opening an issue.
like : 
+ version
+ operating system
+ configs

Or a reproducible config/setup so that people can see for them selfs that there is indeed an issue.
First have you tried running the same file but with out your config?
`neovide -u NONE` 
If the issue doesn't show up in the no config neovide then there is an issue with your config and not neovide.
